device/alpaka/CMakeLists.txt

```diff
@@ -15,16 +15,27 @@ if(alpaka_ACC_GPU_CUDA_ENABLE)
 
   set_source_files_properties(src/utils/make_prefix_sum_buff.cpp PROPERTIES LANGUAGE CUDA)
   set_source_files_properties(src/seeding/spacepoint_binning.cpp PROPERTIES LANGUAGE CUDA)
+
+  set_source_files_properties(src/seeding/track_params_estimation.cpp PROPERTIES LANGUAGE CUDA)
+  set_source_files_properties(src/seeding/seed_finding.cpp PROPERTIES LANGUAGE CUDA)
+  set_source_files_properties(src/seeding/seeding_algorithm.cpp PROPERTIES LANGUAGE CUDA)
```

Comment:
As long as we have a number of files here, it would be a bit more compact to write this as:

```cmake
  set_source_files_properties(
    src/utils/make_prefix_sum_buff.cpp
    src/seeding/spacepoint_binning.cpp
    src/seeding/track_params_estimation.cpp
    src/seeding/seed_finding.cpp
    src/seeding/seeding_algorithm.cpp
    PROPERTIES LANGUAGE CUDA)
```

Reply:
For now, I've swapped this to utilise the alpaka CMake functions: 

https://github.com/alpaka-group/alpaka/blob/develop/cmake/addExecutable.cmake

https://github.com/alpaka-group/alpaka/blob/develop/cmake/addLibrary.cmake

I'm not necessarily 100% sure with that decision though.

It does mean we remove this annoying block of CMake (made more annoying when you consider that there also needs to be an equivalent HIP block), and it guarantees that all the relevant Alpaka CMake flags are kept and applied....

But it duplicates the `traccc-functions` code, which does not seem ideal, but I'm not necessarily sure what the best other approach is? I guess I could add a smaller Alpaka specific CMake file that basically duplicates the functionality of the alpaka funcs, that we call in our CMake, around the existing `traccc_add_executable` etc.